src/mcu.h

```diff
@@ -174,6 +178,17 @@ enum {
     VECTOR_INTERNAL_INTERRUPT_E0, // ADI
 };
 
+static const int ROM1_SIZE = 0x8000;
+static const int ROM2_SIZE = 0x80000;
+static const int RAM_SIZE = 0x400;
+static const int SRAM_SIZE = 0x8000;
+static const int NVRAM_SIZE = 0x8000; // JV880 only
+static const int CARDRAM_SIZE = 0x8000; // JV880 only
+static const int ROMSM_SIZE = 0x1000;
```

Comment:
Minor: could be `static constexpr uint32_t`. Using `uint32_t` for hex addresses probably is a bit more correct.

Reply:
Agreed, but I just moved these constants from elsewhere. I didn't change anything that didn't need to be.